docker-compose.yml

```diff
@@ -5,11 +5,17 @@ services:
     image: postgres:10.1-alpine
     volumes:
       - postgres_data:/var/lib/postgresql/data/
+    ports:
+      - 5432:5432
```

Comment:
Tagging as a potential remnant from development, doesn't look related to other changes.